setup.py

```diff
@@ -3,10 +3,10 @@
 with open("README.rst", "r") as fh:
     long_description = fh.read()
 setuptools.setup(
-    name="moyasar-python",
+    name="moyasar",
     version="0.6",
```

Comment:
we use [semantic versioning](https://semver.org/) so it's three part versioning process. so it'll be `0.6.0`